hkmc2/shared/src/test/mlscript/basics/CyclicModuleForwarders.mls

```diff
@@ -1,14 +1,12 @@
 :js
 
-
-fun f = M.List.map
-module M with
-  fun List: module M2.List = M2.List
+fun f: module M2 = M1.List
+module M1 with
+  fun List: module M2 = M2.List
 module M2 with
-  fun List: module M.List = M.List
+  fun List: module M1 = M1.List
 
-:re
+:todo
+:e
 f
 //│ ═══[RUNTIME ERROR] RangeError: Maximum call stack size exceeded
-
-
```

Comment:
Y u hate space

Reply:
Maybe I should write a script with MLscript to automatically check the white spaces removed before committing🤦